src/cascadia/TerminalSettingsEditor/SettingContainerStyle.xaml

```diff
@@ -0,0 +1,72 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under
+the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.Terminal.Settings.Editor"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <Style x:Key="SettingContainerHeaderStyle" TargetType="StackPanel">
+        <Setter Property="Orientation" Value="Horizontal"/>
+        <Setter Property="Margin" Value="0,0,0,4"/>
+    </Style>
+
+    <Style x:Key="SettingContainerResetButtonStyle" TargetType="Button">
+        <Setter Property="Margin" Value="5,0,0,0"/>
+        <Setter Property="Height" Value="19"/>
+        <Setter Property="Width" Value="19"/>
+        <Setter Property="Padding" Value="0"/>
+    </Style>
+
+    <Style x:Key="SettingContainerFontIconStyle" TargetType="FontIcon">
+        <Setter Property="Foreground" Value="{StaticResource SystemAccentColor}"/>
+        <Setter Property="FontSize" Value="14"/>
+        <Setter Property="FontFamily" Value="Segoe MDL2 Assets"/>
+    </Style>
+
+    <Style x:Key="SettingContainerOverrideTextStyle" TargetType="TextBlock">
+        <Setter Property="FontStyle" Value="Italic"/>
+    </Style>
+
+    <Style TargetType="local:SettingContainer">
+        <Setter Property="Margin" Value="0,24,0,0"/>
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="local:SettingContainer">
+                    <StackPanel>
+
+                        <!--Header space-->
+                        <StackPanel Style="{StaticResource SettingContainerHeaderStyle}">
+                            <TextBlock Text="{TemplateBinding HeaderText}"/>
+                            <Button x:Name="ResetButton"
+                                    Style="{StaticResource SettingContainerResetButtonStyle}">
+
+                                <Button.Content>
+                                    <FontIcon Glyph="&#xE845;"
+                                              Style="{StaticResource SettingContainerFontIconStyle}"/>
+                                </Button.Content>
+
+                                <Button.Background>
+                                    <SolidColorBrush Opacity="0"/>
+                                </Button.Background>

```

Comment:
this seems more complicated than `Background="Transparent"`. Does it have to be?

Reply:
Nice catch! Didn't even know that was an option!